classes/04class/exercises/c04-iac01/mail2kvanitha/iam.tf

```diff
@@ -0,0 +1,62 @@
+resource "aws_iam_role" "class04-iam-role" {
+  name = "class04-iam-role"
+  path = "/"
+
+  assume_role_policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": "sts:AssumeRole",
+            "Principal": {
+               "Service": "ec2.amazonaws.com"
+            },
+            "Effect": "Allow",
+            "Sid": ""
+        }
+    ]
+}
```

Comment:
An option for consuming external files https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file